wavedump/src/wavedump.c

```diff
@@ -1590,9 +1619,6 @@ int main(int argc, char *argv[])
     //int TriggerEdge;
     //CAEN_DGTZ_IOLevel_t FPIOtype;
 
-    /* Set to enable external triggers, so we can trigger on the laser if we
-     * want. */
-    WDcfg.ExtTriggerMode = CAEN_DGTZ_TRGMODE_ACQ_ONLY;

```

Comment:
Should we set this variable?

Reply:
It does successfully enable external triggers if we set it, but I thought it would be better to keep all the trigger stuff in one place, which is where we write to the registers before the acquisition loop.

Reply:
Is this line in the default settings part though? If so, will it just get automatically overwritten? If that's the case it might be nice to have the default settings function set all available parameters just so it's easy to go look at them.

Reply:
It used to be in the default settings function, but that's why I deleted it because it would get overwritten. What I can do is pass a pointer to the boolean I use to flag the trigger type to the default settings method, and set it to a default value there.

Reply:
Right now I used 3 booleans to flag the type of trigger, one for each type: self, external, and software. I should really consolidate them into one string or int variable like:
`
/* 0 for software, 1 for external, 2 for self */
int trig_type = <command line input>;
`

Reply:
Ok, yeah I think it's fine to leave it in the default settings then since I think that should be a place where some normal defaults get written and then later in the code they can be customized.